CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [4.19.0] - 2020-12-07
+## Added
+- Added `create_reservation_seed_from_finances()` to support new `/v1/reservations-seed` functionality
+
```

Comment:
Do many projects have a changelog like this to be aware of? I just want to be aware if I ever branch out from cms-web.

Reply:
We're using the changelog here since this one is a publicly accessible SDK - the internal documentation tools wouldn't really work :)